build_test.go

```diff
@@ -127,14 +129,15 @@ func testBuild(t *testing.T, context spec.G, it spec.S) {
 			})
 			Expect(err).NotTo(HaveOccurred())
 
-			// I don't think there's any way to guess this or apply a matcher instead of checking equality
-			layer.Path = result.Layers[0].Path
+			// The result layer path is autogenerated, so exact equality will fail
+			Expect(result.Layers[0].Path).To(MatchRegexp(".*port-chooser$"))
+			portChooserLayer.Path = result.Layers[0].Path
```

Comment:
Since you've set up `layersDir` and that's used in the build you can do:

```
Expect(result.Layers[0].Path).To(Equal(filepath.Join(layersDir, "port-chooser")))
```